README.md

```diff
@@ -1,10 +1,58 @@
 ![Nodezoo](https://raw.githubusercontent.com/rjrodger/nodezoo-web/to-redux/client/assets/img/logo-nodezoo.png)
 
 # nodezoo-info
-nodezoo.com micro-service handling module. 
+nodezoo.com micro-service handling module.
+One of the micro-services that implement
```

Comment:
yes sorry saw it just now that its was wrong :)

On Mon, Feb 1, 2016 at 12:08 PM, Dean McDonnell <notifications@github.com>
wrote:

> In README.md
> <https://github.com/rjrodger/nodezoo-info/pull/12#discussion_r51410643>:
>
> > @@ -1,10 +1,58 @@
> >  ![Nodezoo](https://raw.githubusercontent.com/rjrodger/nodezoo-web/to-redux/client/assets/img/logo-nodezoo.png)
> >
> >  # nodezoo-info
> > -nodezoo.com micro-service handling module.
> > +nodezoo.com micro-service handling module.
> > +One of the micro-services that implement
>
> Line 5 - 24 remove
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rjrodger/nodezoo-info/pull/12/files#r51410643>.
>